sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -223,15 +222,14 @@ def test_adjusted_mutual_info_score():
     assert_almost_equal(emi, 0.15042, 5)
     # Adjusted mutual information
     ami = adjusted_mutual_info_score(labels_a, labels_b)
-    assert_almost_equal(ami, 0.27502, 5)
+    assert_almost_equal(ami, 0.27821, 5)
```

Comment:
I comment here but this is for the decorator above. Could you remove the line

```python
@ignore_warnings(category=FutureWarning)
```

on the top of the test. The test should not raise anymore warning.